music21/romanText/tsvConverter.py

```diff
@@ -227,15 +235,9 @@ def _changeRepresentation(self):
         >>> tabCd.representationType
         'DCML'
 
-        >>> tabCd.numeral
-        '#vii'
-
         >>> tabCd._changeRepresentation()
         >>> tabCd.representationType
         'm21'
-
-        >>> tabCd.numeral
-        'vii'
```

Comment:
without these tests, it's hard to see what representationType does -- is there a demonstration that shows what DCML vs m21 type does?

Reply:
I removed them because after using `roman.Minor67Default.FLAT`, tabCd.numeral is now '#vii' in both cases, so it seemed trivial. I have replaced it with a more up-to-date test.

Also, I just realized that this means some lines in the docstring text above are now out of date so I revised them.